### DISCUSSION TITLE
Text for coupon code entry there is no option to translate in the latest version of Elementor PRO

### DISCUSSION THREAD

**Discussion by jalarcon666** [elementor/elementor#22931] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Text for coupon code entry there is no option to translate in the latest version of Elementor PRO in cart Widget.( wp-content/plugins/elementor-pro/modules/woocommerce/widgets/cart.php line 2334)

### Describe the solution you'd like

Allow edit in Elementor <input type="text" name="coupon_code" class="input-text" id="coupon_code" value="" placeholder="<?php esc_attr_e( 'Coupon code', 'elementor-pro' ); ?>" /> (wp-content/plugins/elementor-pro/modules/woocommerce/widgets/cart.php line 2334)

### Additional context

You can edit de text of the button "Apply Coupon" but not the input text.

![image](https://github.com/elementor/elementor/assets/88532975/73e730b9-469a-4d9d-842d-f38a57ea96f4)

**Comment by betagoo** [comment]
I have the same problem. There is no translation for Polish language.

**Comment by marcellosa** [comment]
Same issue here. How can we fix that properly? Thank you!

**Comment by razvancuceu** [comment]
Same issue here, how can this be fixed?
